### PR TITLE
49 style news page

### PR DESCRIPTION
**Pull request by daphne-simons**
## Definition of Done 

For every submission to the codebase, check that it meets the DoD below.

* [x] Code fulfils the user story and scenarios set out in the github task and UI prototype.
* [x] The feature has been thoroughly manually tested. (Ask other devs to give some fresh eyes on this as you go.)
* [x] All tests run and passed from terminal.  Any broken tests fixed, not disabled or skipped.
> `npm test`
* [ ] Tests added or updated where appropriate.  (Be pragmatic about adding new tests, think; how critical is this module or function? If this function broke due to changes, how would it affect the user experience?)
* [ ] eslint run and passes. Any issues fixed.
> `npm run lint`
* [ ] PR created in Github.
* [ ] Two peer reviews from other devs.
* [ ] One peer review from one of the teachers.
* [ ] After all the peer review comments are addressed, the branch is merged to main!

### CODE REVIEW

File: client/subcomponents/News/News.jsx [hunk 1]
```diff
@@ -5,13 +5,33 @@ export default function News({ news }) {
...
   return (
-    <article>
-      <h2>{news.title}</h2>
-      <h2>
-        By {news.firstName} {news.lastName}:
-      </h2>
-      <p>{news.content}</p>
-      <p>{createTime}</p>
-    </article>
+    <div className="relative rounded-3xl border-3 shadow-xl">
```
Comment (johnny-sainsbury) [inline client/subcomponents/News/News.jsx]: do some research into "semantic html" and see where you might be able to add more to this component. you've already put it to great use here already, but overall less `<div>` use and more specifically labeled elements means this code base will be more readable and more accessible for users :) [semantic html guide](https://www.freecodecamp.org/news/semantic-html5-elements/#:~:text=Semantic%20HTML%20elements%20are%20those,content%20that%20is%20inside%20them.)

Comment (johnny-sainsbury) [inline client/subcomponents/News/News.jsx]: don't forget to apply mobile styling to this component! looks like padding/margin adjustments and column numbers should do the trick

**Review by johnny-sainsbury — COMMENTED** [review]
hey great work here! just a few changes I've left in comments for you. once these are done you won't need to open a new PR, just drag the ticket back into "for review" :)

also please do a "git pull" on the dev branch before pushing back up, it looks like there are conflicts currently!

File: client/subcomponents/News/News.jsx (hunk 1 above)
Comment (daphne-simons) [reply client/subcomponents/News/News.jsx]: Less divs, more sections. Articles too!  
Snappy for the screen readers. Hope you like it. Thanks Johnny.

Comment (daphne-simons) [reply client/subcomponents/News/News.jsx]: Thanks, added these as well, yeah you're right padding does the work! Every time.

**Review by johnny-sainsbury — APPROVED** [review]

**Pull request merged**